src/main/java/online/book/store/exception/CustomGlobalExceptionHandler.java

```diff
@@ -41,6 +41,12 @@ public ResponseEntity<Object> handleEntityNotFoundException(EntityNotFoundExcept
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(RegistrationException ex) {
+        Map<String, Object> body = Map.of("error", ex.getMessage());
+        return new ResponseEntity<>(body, HttpStatus.BAD_REQUEST);
```

Comment:
It's pretty same as a body in handleEntityNotFoundException method, let's avoid code duplication.